backend/src/main/java/com/example/demo/service/FirebaseService.java

```diff
@@ -17,7 +17,7 @@
 
     public String savePasteDetails(Paste paste) throws InterruptedException, ExecutionException {
         Firestore dbFirestore = FirestoreClient.getFirestore();
-        ApiFuture<WriteResult> collectionsApiFuture = dbFirestore.collection("pastes").document(paste.getId()).set(paste);
+        ApiFuture<WriteResult> collectionsApiFuture = dbFirestore.collection("pastes").document(paste.getTitle()).set(paste);
```

Comment:
Follow https://firebase.google.com/docs/firestore/manage-data/add-data#java_5 to add document with ID generated by Firestore. Return the ID instead of the update time.